apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -56,12 +56,26 @@ pub(crate) struct Conf {
     /// Logging configuration
     #[serde(rename = "experimental_logging", default)]
     pub(crate) logging: Logging,
+
+    #[cfg(feature = "telemetry_next")]
+    #[serde(rename = "logging", default)]
+    pub(crate) new_logging: config_new::logging::Logging,
     /// Metrics configuration
     pub(crate) metrics: Metrics,
     /// Tracing configuration
     pub(crate) tracing: Tracing,
     /// Apollo reporting configuration
     pub(crate) apollo: apollo::Config,
+
+    #[cfg(feature = "telemetry_next")]
+    /// Event configuration
+    pub(crate) events: config_new::events::Events,
+    #[cfg(feature = "telemetry_next")]
+    /// Span configuration
+    pub(crate) spans: config_new::spans::Spans,
+    #[cfg(feature = "telemetry_next")]
+    /// Instrument configuration
+    pub(crate) instruments: config_new::instruments::Instruments,
```

Comment:
It looks like it's only related to metrics, so same question why not putting that in `metrics` ?

Reply:
[9ac775a](https://github.com/apollographql/router/pull/4061/commits/9ac775a02e509a68dec99653ce010d109eab098e)

Reply:
For consistency with spans and events this is at the same level. But let's discuss.